ui/components/ui/custom/custom-input.tsx

```diff
@@ -91,20 +91,33 @@ export const CustomInput = <T extends FieldValues>({
       render={({ field }) => (
         <>
           <FormControl>
-            <Input
-              id={name}
-              isRequired={inputIsRequired}
-              label={inputLabel}
-              labelPlacement={labelPlacement}
-              placeholder={inputPlaceholder}
-              type={inputType}
-              variant={variant}
-              size={size}
-              isInvalid={isInvalid}
-              defaultValue={defaultValue}
-              endContent={endContent}
-              {...field}
-            />
+            {type === "textarea" ? (
```

Comment:
@MrCloudSec Thanks for the contribution. 

We're currently using this component 36 times across 15 different files. It's likely the most important component in Prowler v5 (it's present in every form we have). During our discussion today, I briefly explained how it works.

The <input> element is an HTML component, while <textarea> is a completely distinct HTML component. By making changes to CustomInput, you’re merging two separate components into one, which is not a viable approach.

To summarize, we would need to create a CustomTextArea component and customize it in the same way as CustomInput, including all the necessary props so it can be reused consistently without breaking anything.

I'm happy to assist with any changes needed to move forward, but this will have to wait until the release is finished as there are higher priority items

Please ensure this file is not modified in this PR.

Thank you.